examples/connectivity/plot_ica_resting_state.py

```diff
@@ -13,6 +13,11 @@
 nyu_dataset = datasets.fetch_nyu_rest(n_subjects=1)
 func_filename = nyu_dataset.func[0]
 
+# print basic information on the dataset
+print('First anatomical nifti image (3D) is at: %s' % nyu_dataset.anat_anon[0])
+print('First functional nifti image (4D) is at: %s' %
+      nyu_dataset.func[0])  # 4D data
```

Comment:
> I don't think it should say first here since the list is of size 1, maybe something like 'Functional 4D image is at: '

This is where my earlier comment was supposed to end up.